src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+  const [people, setPeople] = useState<Person[]>([]);
+
+  useEffect(() => {
+    setIsLoading(true);
+    setIsError(false);
+    getPeople()
+      .then(setPeople)
+      .catch(() => setIsError(true))
+      .finally(() => setIsLoading(false));
+  }, []);
+
+  return (
+    <>
+      <h1 className="title">People Page</h1>
+
+      <div className="block">
+        <div className="box table-container">
+          {isLoading && <Loader />}
+
+          {isError && !isLoading && (
+            <p data-cy="peopleLoadingError" className="has-text-danger">
+              Something went wrong
+            </p>
+          )}
+
+          {people.length === 0 && !isLoading && (
+            <p data-cy="noPeopleMessage">There are no people on the server</p>
+          )}
+
+          {people.length !== 0 && <PeopleTable people={people} />}
```

Comment:
This is a code quality issue. You are checking `people.length !== 0` before rendering `PeopleTable`. Instead of checking for length, you could simply check for `people` because an empty array is considered `falsy` in JavaScript. This would make your code more concise.